numpy/lib/recfunctions.py

```diff
@@ -528,12 +528,12 @@ def drop_fields(base, drop_names, usemask=True, asrecarray=False):
     >>> from numpy.lib import recfunctions as rfn
     >>> a = np.array([(1, (2, 3.0)), (4, (5, 6.0))],
     ...   dtype=[('a', np.int64), ('b', [('ba', np.double), ('bb', np.int64)])])
-    >>> rfn.drop_fields(a, 'a')
+    >>> rfn.drop_fields(a, 'a', usemask=False)
     array([((2., 3),), ((5., 6),)],
           dtype=[('b', [('ba', '<f8'), ('bb', '<i8')])])
-    >>> rfn.drop_fields(a, 'ba')
+    >>> rfn.drop_fields(a, 'ba', usemask=False)
     array([(1, (3,)), (4, (6,))], dtype=[('a', '<i8'), ('b', [('bb', '<i8')])])
-    >>> rfn.drop_fields(a, ['ba', 'bb'])
+    >>> rfn.drop_fields(a, ['ba', 'bb'], usemask=False)
```

Comment:
If you have to change the example for it to pass, that means you are changing the behavior for non-masked arrays drastically.  That is a no-go.  I am not sure how `recfunctions` work with `usemask=True` usually; not sure the option really makes much sense here to begin with (the input is masked or not, I don't know why we need an argument).